### DISCUSSION TITLE
My selected school has incorrect or incomplete information e.g. domains or campus location

### DISCUSSION THREAD

**Discussion by hadiulofficial** [community/community#143244] [GitHub Education]
### Hi everyone,

I'm near campus locations, but I'm getting rejected multiple times for the following reason:

You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect (e.g., missing campus location(s)), please select the ‘My selected school has incorrect or incomplete information (e.g., domains or campus locations)’ option when submitting a GitHub Education support ticket

**Comment by THE-DEEPDAS** [comment]
See bro this is an issue i also saw the first time so you can do the following things.
The rejection message you’re receiving suggests that GitHub Education's verification system cannot confirm your physical proximity to your selected campus, which is essential for account verification. Here are steps that might help resolve the issue:

Allow Location Access: Make sure both your operating system and browser are set to allow GitHub to access your location data. In most cases, you’ll be prompted to approve location access when you visit the site, but it’s worth checking your privacy settings to ensure GitHub isn’t being blocked.

Disable VPNs: GitHub’s system can mistakenly interpret a VPN as masking your true location. If you have any VPNs or privacy proxies enabled, try disabling them temporarily during the verification process.

Documentation for Remote Learning: If you’re enrolled in virtual or remote classes, try providing additional documentation (like a class schedule or enrollment letter) explicitly showing your remote status. GitHub Education might then be able to accept this as proof if they can’t detect your campus location.

Report Location Issues: If your campus location isn’t being recognized, go to the “My selected school has incorrect or incomplete information” option when submitting a support ticket to clarify any issues related to your school’s campus information.
